documentation/site/pages/provers/quick-start.mdx

```diff
@@ -57,75 +145,136 @@ Or remove all volumes and data from the service:
 
-## Sending a Test Proof to Bento
+### Multi GPU
 
-### Bento CLI installation
+Under the `exec_agent` service in `compose.yml`, the default configuration utilises a single GPU:
+
+```yml [compose.yml]
+deploy:
+  resources:
+    reservations:
+      devices:
+        - driver: nvidia
+          device_ids: ['0']
+          capabilities: [gpu]
+
+# gpu_prove_agent1:
+#   <<: *agent-common
+#   runtime: nvidia
+#   deploy:
+#     resources:
+#       reservations:
+#         devices:
+#           - driver: nvidia
+#             device_ids: ['1']
+#             capabilities: [gpu]
+```
 
-To install the latest CLI version:
+To add a second GPU, first check your GPUs are recognised with:
 
 ```sh [Terminal]
-cargo install --git https://github.com/risc0/risc0 bento-client --bin bento_cli
+nvidia-smi -L
 ```
+which should output something like:
 
-### Using the Bento CLI
+```sh
+GPU 0: NVIDIA GeForce RTX 3090 (UUID: GPU-abcde123-4567-8901-2345-abcdef678901)
+GPU 1: NVIDIA GeForce RTX 3090 (UUID: GPU-fedcb987-6543-2109-8765-abcdef123456)
+```
 
-Once everything is up, you can send a test proof to Bento to see if everything works. There are two options for test proofs:
+We can see that GPU 1 is listed with the device ID of `1`. To add this GPU, uncomment `gpu_prove_agent1` from `compose.yml`:
+
+```yml [compose.yml]
+gpu_prove_agent1: # [!code hl]
+  <<: *agent-common
+  runtime: nvidia
+  deploy:
+    resources:
+      reservations:
+        devices:
+          - driver: nvidia
+            device_ids: ['1'] # [!code hl]
+            capabilities: [gpu]
+```
 
-1. The Bento CLI allows for a simple test vector with a specified cycle count via the `-c` flag:
+For 3 or more GPUs, add the corresponding `gpu_prove_agentX` where `X` matches the device ID of each GPU, making sure that the `device_ids` field is populated with a matching `X`: `['X']`.
 
-```sh [Terminal]
-RUST_LOG=info bento_cli -c 32
-```
+### Segment Size
 
-2. The Bento CLI also accepts [existing program and input files](/developers/tutorials/build):
+Under the `exec_agent` service in `compose.yml` you can configure the segment size; it defaults to 21:
 
-```sh [Terminal]
-RUST_LOG=info bento_cli -f ./crates/bento-client/method_name -i /tmp/input.bin
+```yml [compose.yml]
+exec_agent:
+  <<: *agent-common
+  runtime: nvidia
+
+  mem_limit: 4G
+  cpus: 4
+
+  environment:
+    <<: *base-environment
+    RISC0_KECCAK_PO2: ${RISC0_KECCAK_PO2:-17}
+    entrypoint: /app/agent -t exec --segment-po2 ${SEGMENT_SIZE:-21} # [!code hl]
 ```
 
-## Depositing Funds to the Market
+`SEGMENT_SIZE` is specified in powers-of-two (po2). Larger segment sizes are preferable for performance, but require more GPU VRAM. To pick the right `SEGMENT_SIZE` for your GPU VRAM, see [the performance optimization page](/provers/performance-optimization#finding-the-maximum-segment_size-for-gpu-vram).
```

Comment:
The wording here makes it seem like you need to set this value in the compose file manually, you actually just need to set the env variable `SEGMENT_SIZE` and this will override this default of 21.